packages/pyright-internal/src/analyzer/packageTypeVerifier.ts

```diff
@@ -430,7 +430,17 @@ export class PackageTypeVerifier {
         const dirEntries = this._fileSystem.readdirEntriesSync(dirPath);
 
         dirEntries.forEach((entry) => {
-            if (entry.isFile()) {
+            let isFile = entry.isFile();
```

Comment:
We could just call pathUtils.getFileSystemEntries here rather than duplicating this logic.